docs/guide/known-limitations.md

```diff
@@ -34,4 +34,5 @@ you can't compare the arguments in a formula like this:
   * Asynchronous functions
   * Structured references ("Tables")
   * Currency data type
-  * Relative named expressions
\ No newline at end of file
+  * Relative named expressions
+* We immediately instantiate references to single cells to their values instead of treating them as 1-length ranges, which slightly changes behavior of some functions (e.g. NPV).
```

Comment:
```suggestion
  * Relative named expressions
  * We immediately instantiate references to single cells to their values instead of treating them as 1-length ranges, which slightly changes behavior of some functions (e.g. NPV).
```